lib/formatters/srt_formatter.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Subtitles.SrtFormatter do
+  def format(subtitles) do
+    (subtitles |> Enum.with_index(1) |> Enum.map(&build(&1)) |> Enum.join("\n\n"))
+    <> "\n\n"
```

Comment:
I don't now if i prefer this instead: 
```
iex(3)> [1,2,3] ++ [nil] |> Enum.join("\n\n")
"1\n\n2\n\n3\n\n"
```

Maybe not.

Reply:
I think that is about as convoluted as before, so trailing \n\n as string is probably best

Reply:
One thing you could do would be to extract and explain the behaviour as a funktion that you could pipe into.

```
.... Enum.join() |> add_closing_new_lines
```

would then be a oneliner:
```
defp add_closing_new_lines(srt_string), do: srt_string <> "\n\n"
```

Reply:
There we go! Will add.